modules/statics/src/coins.ts

```diff
@@ -1117,4 +1119,24 @@ export const coins = CoinMap.fromCoins([
   teosToken('teos:IQ', 'Everipedia', 3, 'testtoken112', UnderlyingAsset.IQ, AccountCoin.DEFAULT_FEATURES, '', 'IQ'),
   teosToken('teos:BOX', 'Box', 6, 'kvszn1xyz1bu', UnderlyingAsset.EOS_BOX, AccountCoin.DEFAULT_FEATURES, '', 'BOX'),
   teosToken('teos:USDT', 'Tether', 4, 'lionteste212', UnderlyingAsset.USDT, AccountCoin.DEFAULT_FEATURES, '', 'USDT'),
+  tavaxErc20('Qi', 'BenQi', 18, '0x8729438eb15e2c8b576fcc6aecda6a148776c0f5', UnderlyingAsset.QI),
+  tavaxErc20('XAVA', 'Avalaunch', 18, '0xd1c3f94de7e5b45fa4edbba472491a9f4b166fc4', UnderlyingAsset.XAVA),
+  tavaxErc20('KLO', 'Kalao', 18, '0xb27c8941a7df8958a1778c0259f76d1f8b711c35', UnderlyingAsset.KLO),
+  tavaxErc20('JOE', 'Trader Joe', 18, '0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd', UnderlyingAsset.JOE),
```

Comment:
I see that same address works for testnet and mainnet.
Should we add mainnet token also?
https://avascan.info/blockchain/c/address/0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd
https://testnet.avascan.info/blockchain/c/address/0x6e84a6216ea6dacc71ee8e6b0a5b7322eebc0fdd

Reply:
Does the definition on mainnet have any impact on production now?
My concern if it is a way to enable/disable tokens in production until they are fully supported (all the development done)